test/client_test.exs

```diff
@@ -2,7 +2,6 @@ defmodule ClientTest do
   use ExUnit.Case, async: false
   alias MLLP.Client
   alias MLLP.Client.Error
-  doctest Client
```

Comment:
I'm not sure why this was ever doc tested, but it's not something that can effectively be doc tested (i.e., it's  impure).